torchci/lib/drciUtils.ts

```diff
@@ -24,14 +29,73 @@ Note: Links to docs will display an error until the docs builds have been comple
 }
 
 export function formDrciComment(
-    pr_num: number,
-    pr_results: string = "",
+  pr_num: number,
+  pr_results: string = "",
+  sevs: string = ""
 ): string {
-    const header = formDrciHeader(pr_num);
-    const comment =
-        `${DRCI_COMMENT_START}
+  const header = formDrciHeader(pr_num);
+  const comment = `${DRCI_COMMENT_START}
 ${header}
+${sevs}
 ${pr_results}
 ${DRCI_COMMENT_END}`;
-    return comment;
+  return comment;
+}
+
+
+export async function getDrciComment(
+  octokit: Octokit,
+  owner: string,
+  repo: string,
+  prNum: number
+): Promise<{ id: number; body: string }> {
+  const commentsRes = await octokit.rest.issues.listComments({
+    owner: owner,
+    repo: repo,
+    issue_number: prNum,
+  });
+  for (const comment of commentsRes.data) {
+    if (comment.body!.includes(DRCI_COMMENT_START)) {
+      return { id: comment.id, body: comment.body! };
+    }
+  }
+  return { id: 0, body: "" };
+}
+
+export function getActiveSEVs(issues: IssueData[]): string {
+  const activeSEVs = issues.filter(
+    (issue: IssueData) => issue.state === "open"
+  );
+  if (activeSEVs.length === 0) {
+    return "";
+  }
+  const [merge_blocking, not_merge_blocking] = _.partition(
+    activeSEVs,
+    (issue: IssueData) => issue.body.toLowerCase().includes("merge blocking")
+  );
+  const sev_list = merge_blocking
+    .concat(not_merge_blocking)
+    .map(
+      (issue: IssueData) =>
+        `* ${
+          issue.body.toLowerCase().includes("merge blocking")
+            ? "(merge blocking) "
+            : ""
+        }[${issue.title}](${issue.html_url.replace(
+          "github.com",
+          "hud.pytorch.org"
+        )})`
+    )
+    .join("\n");
+  if (merge_blocking.length > 0) {
+    return `## :heavy_exclamation_mark: ${merge_blocking.length} Merge Blocking SEVs
+There is an active merge blocking SEV.  Please view them [here](${SEV_SEARCH_URL}) or below:
+${sev_list}\n
+If you must merge, use \`@pytorchbot merge -f\`.`;
+  } else {
+    return `## :heavy_exclamation_mark: ${activeSEVs.length} Active SEVs
+There are ${activeSEVs.length} currently active SEVs.   If your PR is affected, please view them [here](${SEV_SEARCH_URL}) or below:
```

Comment:
Nit: `please view them [here](${SEV_SEARCH_URL}) or below` sound weird because both are "here" IMO.  May be we can just stick to the sev list from Rockset because that's the one that drive Dr. CI sev section anyway. (Then we can also remove SEV_SEARCH_URL I think)

Reply:
removed